pmd-rules.xml

```diff
@@ -30,6 +30,10 @@
         <exclude name="DataClass" />
         <exclude name="ExcessiveImports" />
         <exclude name="AvoidUncheckedExceptionsInSignatures" />
+        <exclude name="GodClass" /> <!-- Should consider fixing this -->
```

Comment:
This xml file is affecting all files in the repo. If you really want to suppress something because of this patch then use annotations "@SuppressWarnings"

Reply:
Done